src/StateNode.ts

```diff
@@ -1276,7 +1281,7 @@ class StateNode<
         : currentState
         ? currentState.meta
         : undefined,
-      events: resolvedStateValue ? (raisedEvents as TEvent[]) : [],
```

Comment:
this has always been empty, as we empty `raisedEvents` (by mutating it) before we reach this point